contactaccessrights/CRM/Contactaccessrights/Test/Fabricator/Rights.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * Fabricates rights for tests.
+ *
+ * @author Camilo Rodriguez - camilo@compucorp.co.uk
+ */
+abstract class CRM_Contactaccessrights_Test_Fabricator_Rights {

```

Comment:
Sorry, but none of the existing fabricators are abstract classes. Only [BaseAPIFabricator](https://github.com/civicrm/civihr/blob/staging/hrjobcontract/CRM/Hrjobcontract/Test/Fabricator/BaseAPIFabricator.php) is abstract, but that's because it's just a template for the other fabricators.
Using abstract to not allow a class to be instantiated is conceptually wrong. That is not what abstract classes are for